CONTRIBUTING.md

```diff
@@ -263,8 +198,8 @@ This is the slowest part of package updates. If any packages are skipped during
 
 Before developing you'll need to install:
 * [NodeJS and NPM](https://nodejs.org/)
-* [Docker](https://docs.docker.com/desktop/)
-  * Alternatively, you can try [Podman](https://github.com/containers/podman). [Homebrew](https://brew.sh/) can be used for the install `$ brew install podman`
+* [podman desktop](https://podman-desktop.io/)
+  * Alternatively, you can try [Docker](https://docs.docker.com/desktop/).
```

Comment:
```suggestion
```

No support anywhere for Docker Desktop. 🚫

Reply:
k, this PR is 2 of 4... the podman/docker work is here #345... I'll apply the comments and remove docker completely

Reply:
see #345 for resolve